FreshAir/UI/RZFUpdateViewModel.m

```diff
@@ -9,16 +9,44 @@
 #import "RZFUpdateViewModel.h"
 #import "RZFReleaseNotes.h"
 
-static NSString *RZFLocalizedValue(NSBundle *bundle, NSString *key) {
-    NSString *fullKey = [@"freshair.update." stringByAppendingString:key];
-    NSString *value = [[NSBundle mainBundle] localizedStringForKey:fullKey value:nil table:@"FreshAirUpdate"];
-    if ([value isEqual:fullKey]) {
-        value = [[NSBundle mainBundle] localizedStringForKey:fullKey value:nil table:nil];
+static NSString * const RZFUpdateViewModelForcedUpdatePrefix = @"freshair.update.forced";
+static NSString * const RZFUpdateViewModelUpdatePrefix = @"freshair.update";
+static NSString * const RZFUpdateViewModelStringTableName = @"FreshAirUpdate";
+static NSString * const RZFUpdateViewModelUpdateImageName = @"freshair_update";
+
+static NSString * const RZFUpdateViewModelLocalizedKeyTitle = @"title";
+static NSString * const RZFUpdateViewModelLocalizedKeyDescription = @"description";
+static NSString * const RZFUpdateViewModelLocalizedKeyConfirm = @"confirm";
+static NSString * const RZFUpdateViewModelLocalizedKeyDismiss = @"dismiss";
+
+static NSString *RZFLocalizedValue(NSString * _Nonnull prefix, NSString * _Nonnull key, NSBundle * _Nullable fallbackBundle, NSString * _Nullable fallbackTableName) {
```

Comment:
I wouldn't pass in the `RZFUpdateViewModelStringTableName`, it's always the same.